students/ChelseaSmith/Lesson3/slicingfunctions.py

```diff
@@ -1,17 +1,17 @@
 def flip_first_last(seq):
-    orig = tuple(seq)
-    new = list(seq)
-    new[0] = orig[-1]
-    new[-1] = orig[0]
+    orig = tuple(seq)  # make a tuple so this stays safe
+    new = list(seq)  # make a list to mess around with
+    new[0] = orig[-1]  # make first value of new list last value of old list
+    new[-1] = orig[0]  # make last value of new list first value of old list
     return new
```

Comment:
this function should only use slicing, not indexing (which is what you have); there is also no need for special type handling or casting - if using slicing it will work for tuples, strings and lists.